sass/custom/_exec.scss

```diff
@@ -1,24 +1,49 @@
+.role {
+    display: flex;
+    align-items: center;
+}
+
+.role>* {
+    margin: 1vh 0;
+}
+
+.role a {
+    margin-left: 0.5vw;
```

Comment:
Please use Bootstrap classes for margins (either add class e.g. `ms-3` or `@extend .ms-3` in here)